src/cli.ts

```diff
@@ -109,6 +128,7 @@ const main = defineCommand({
     uninstall: remove,
     un: remove,
     detect,
+    "detect-runner": detectRunner,
```

Comment:
It would be nice if we simplify to have `nypm detect` that displays both detection results.